<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContremForm</class>
 <widget class="QDialog" name="ContremForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convex Hull Continuum Removal</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="12" column="1" colspan="7">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QToolButton" name="btnROI">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="5">
    <widget class="QLineEdit" name="txtROI"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Spectral Raster</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>ENVI ROI</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Wavelength Map</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1" colspan="7">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnRun">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnHelp">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="3">
    <widget class="QRadioButton" name="rdoSample">
     <property name="text">
      <string>Sample</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="5" column="2" colspan="5">
    <widget class="QLineEdit" name="txtOutput"/>
   </item>
   <item row="0" column="2" colspan="5">
    <widget class="QLineEdit" name="txtSpectra"/>
   </item>
   <item row="4" column="4">
    <widget class="QSpinBox" name="spnWLCol"/>
   </item>
   <item row="4" column="6">
    <widget class="QSpinBox" name="spnBandCol"/>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Buffer Size</string>
     </property>
    </widget>
   </item>
   <item row="14" column="1" colspan="7">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Threads</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Statistics</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="5">
    <widget class="QLineEdit" name="txtWavelength"/>
   </item>
   <item row="6" column="2">
    <widget class="QComboBox" name="cboDriver"/>
   </item>
   <item row="7" column="2" colspan="5">
    <widget class="QDoubleSpinBox" name="spnMinWL">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>99999.990000000005239</double>
     </property>
    </widget>
   </item>
   <item row="8" column="2" colspan="5">
    <widget class="QDoubleSpinBox" name="spnMaxWL">
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>99999.990000000005239</double>
     </property>
    </widget>
   </item>
   <item row="5" column="7">
    <widget class="QToolButton" name="btnOutput">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1" colspan="7">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Max. Wavelength</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2" colspan="5">
    <widget class="QSpinBox" name="spnThreads">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="10" column="2" colspan="5">
    <widget class="QSpinBox" name="spnBuffer">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>4096</number>
     </property>
     <property name="value">
      <number>256</number>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Raster Driver</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QRadioButton" name="rdoPopulation">
     <property name="text">
      <string>Population</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Band Index Col</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QToolButton" name="btnSpectra">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Wavelength Col</string>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="QToolButton" name="btnWavelength">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QCheckBox" name="chkHeader">
     <property name="text">
      <string>Header Row</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Output File</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Min. Wavelength</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
